pom.xml

```diff
@@ -1283,6 +1283,7 @@
                                     <exclude>org.codehaus.plexus:plexus-utils</exclude>
                                     <exclude>com.google.guava:guava</exclude>
                                     <exclude>com.fasterxml.jackson.core:jackson-core</exclude>
+                                    <exclude>com.fasterxml.jackson.core:jackson-databind</exclude>
```

Comment:
We shouldn't need this after updating to the latest Airbase (yet to be released) that updates Jackson.

I'm working on releasing it now and will put up a separate PR to update.